.github/workflows/platform.assignIssueToProject.yml

```diff
@@ -16,17 +16,33 @@ jobs:
     name: Add Issues to Projects
     runs-on: ubuntu-latest
     steps:
-      - name: Add Bug to Bug Board
-        uses: srggrs/assign-one-project-github-action@1.2.1
+      - name: 'Add Bug to Bug Board'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         if: |
           contains(github.event.issue.labels.*.name, 'bug')
-        with:
-          project: https://github.com/orgs/Azure/projects/364
-          column_name: 'Needs triage'
-      - name: Add Issues to Backlog Board
-        uses: srggrs/assign-one-project-github-action@1.2.1
+        shell: pwsh
+        run: |
+          gh api graphql -f query='
+            mutation($project:ID!, $issue:ID!) {
+              addProjectV2ItemById(input: {projectId: $project, contentId: $issue}) {
+                item {
+                  id
+                }
+              }
+            }' -f project="PVT_kwDOAGhwUs4AJa3a" -f issue=${{ github.event.issue.node_id }} --jq '.data.addProjectV2ItemById.projectV2Item.id'
```

Comment:
Are the project IDs fine to be shared here?

Reply:
We could run the code to get the ID. Then only the number of the project is shared, which is also in the URL. That should not be an issue. I will change it today.